commands/show/cluster/command.go

```diff
@@ -482,6 +492,10 @@ func printV5Result(args Arguments, details *models.V5ClusterDetailsResponse,
 		clusterTable = append(clusterTable, formatCredentialDetails(credentialDetails)...)
 	}
 
+	if webUIURL != "" {
```

Comment:
I wouldn't have expected to need these bits of code twice in the same file / command.

Can it be extracted into a function `webUIURL()` ?

Reply:
You mean `webUIURL, _ := webui.ClusterDetailsURL(args.apiEndpoint, details.ID, details.Owner)` ? That's called two times, but with different arguments.